modules/lily-compiler/src/main/java/io/github/tomboyo/lily/compiler/oas/model/IExplode.java

```diff
@@ -0,0 +1,10 @@
+package io.github.tomboyo.lily.compiler.oas.model;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.DEDUCTION;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = DEDUCTION)
+@JsonSubTypes({@JsonSubTypes.Type(None.class), @JsonSubTypes.Type(Explode.class)})
+public sealed interface IExplode permits None, Explode {}
```

Comment:
Explode is just a boolean, right? Should replace this with Optional<Boolean>, then, like we do for other primitives.